index.css

```diff
@@ -0,0 +1,67 @@
+@import url("https://argon-css.netlify.app/components/index.css");
+
+/* header search bar */
+
+.search__bar__container {
+  width: 30rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.search__bar {
+  margin: 0 auto;
+  width: 100%;
+  height: 3rem;
+  padding: 0 1.25rem;
+  font-size: 1rem;
+  border-radius: 1rem;
+  border: none;
+  outline: none;
```

Comment:
Don't make the outline as none. It creates an accessibility issue.
Instead of that use - 
```
outline: 2px solid transparent
outline-offset: 2px
```